packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -1681,7 +1681,7 @@ class FocusManager with DiagnosticableTreeMixin, ChangeNotifier {
       TargetPlatform.linux || TargetPlatform.macOS || TargetPlatform.windows => true,
       TargetPlatform.android || TargetPlatform.fuchsia || TargetPlatform.iOS => false,
     };
-    if (isDesktop || kIsWeb) {
+    if (isDesktop) {
```

Comment:
Instead of skipping all web, can you skip Android web only? Desktop web should be using this.

Reply:
Oh, I guess it would be anyhow.  But let's try and keep iOS too.